### PR TITLE
Update default networks

### PR DESCRIPTION
**Pull request by tolgahan-arikan**

### CODE REVIEW

**Review by pkieltyka — APPROVED** [review]

File: packages/network/src/config.ts [hunk 1]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
...
+    relayer: { url: relayerURL('mainnet') },
+    indexerUrl: indexerURL('mainnet')
+  },
+  {
+    ...networks[ChainId.POLYGON],
+    rpcUrl: nodesURL('polygon'),
+    relayer: { url: relayerURL('polygon') },
+    indexerUrl: indexerURL('polygon'),
+    isDefaultChain: true,
+    isAuthChain: true
```
Comment (attente) [inline packages/network/src/config.ts:301]: Don't know if we want to change the default network from Ethereum to Polygon yet. Can we maintain compatibility with the current release?

File: packages/network/src/config.ts [hunk 2]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
...
+    relayer: { url: relayerURL('rinkeby') },
+    indexerUrl: indexerURL('rinkeby')
+  },
+  {
+    ...networks[ChainId.POLYGON_MUMBAI],
+    rpcUrl: nodesURL('mumbai'),
+    relayer: { url: relayerURL('mumbai') },
+    indexerUrl: indexerURL('mumbai'),
+    isDefaultChain: true,
+    isAuthChain: true
```
Comment (attente) [inline packages/network/src/config.ts:349]: Same question here about switching default from Rinkeby to Mumbai.

File: packages/network/src/config.ts [hunk 3]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
...
-      relayer: { url: urlClean(`${vars.baseRelayerUrl}/rinkeby`) },
-      isDefaultChain: true
-    },
-    {
-      ...networks[ChainId.GOERLI],
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
-      rpcUrl: urlClean(`${vars.baseRpcUrl}/goerli`),
-      relayer: { url: urlClean(`${vars.baseRelayerUrl}/goerli`) },
-      isAuthChain: true
-    }
```
Comment (attente) [inline packages/network/src/config.ts:325]: Seems like we should keep Goerli in the changes too.

File: packages/network/src/config.ts [hunk 4]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
...
+    relayer: { url: relayerURL('optimism') }
   }
-)
+]
 
-export const testnetNetworks = createNetworkConfig(
-  (vars: { [key: string]: any }) => [
-    {
-      ...networks[ChainId.RINKEBY],
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```
Comment (attente) [inline packages/network/src/config.ts:314]: Also missing in the changes.

File: packages/network/src/config.ts [hunk 5]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
...
-      rpcUrl: urlClean(`${vars.baseRpcUrl}/rinkeby`),
-      relayer: { url: urlClean(`${vars.baseRelayerUrl}/rinkeby`) },
-      isDefaultChain: true
-    },
-    {
-      ...networks[ChainId.GOERLI],
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```
Comment (attente) [inline packages/network/src/config.ts:321]: Also missing in the changes.

File: packages/network/src/config.ts [hunk 6]
```diff
@@ -282,51 +285,73 @@ export const networks: Record<ChainId, NetworkConfig> = {
 
 export type ChainIdLike = NetworkConfig | BigNumberish
 
-export type NetworksBuilder = (vars: { [key: string]: any }) => NetworkConfig[]
-
-export const mainnetNetworks = createNetworkConfig(
-  (vars: { [key: string]: any }) => [
-    {
-      ...networks[ChainId.MAINNET],
-      ensAddress: '0x00000000000C2E074eC69A0dFb2997BA6C7d2e1e',
```
[the remark this answers is not in the document]
Comment (tolgahan-arikan) [reply packages/network/src/config.ts:291]: sure, I can add it back but currently we aren't using it in the webapp. what do u think?

File: packages/network/src/config.ts (hunk 1 above)
Comment (tolgahan-arikan) [reply packages/network/src/config.ts:301]: this one as well is configured this way in webapp

https://github.com/0xsequence/wallet-webapp/blob/master/src/stores/NetworkStore.ts#L52

File: packages/network/src/config.ts (hunk 2 above)
Comment (tolgahan-arikan) [reply packages/network/src/config.ts:349]: same as prev answer

File: packages/network/src/config.ts (hunk 6 above)
[the remark this answers is not in the document]
Comment (attente) [reply packages/network/src/config.ts:291]: I think it's best to add it back.

**Review by attente — APPROVED** [review]

**Pull request merged**